frontend/src/components/core/MainMenu/MainMenu.tsx

```diff
@@ -495,28 +517,56 @@ function MainMenu(props: Props): ReactElement {
     }
   }
 
+  const { hostIsOwner } = props
+
+  const showDevelopmentMenu = (() => {
+    if (props.toolbarMode == Config.ToolbarMode.DEVELOPER) {
+      return true
+    } else if (
+      props.toolbarMode == Config.ToolbarMode.VIEWER ||
+      props.toolbarMode == Config.ToolbarMode.MINIMAL
+    ) {
+      return false
+    } else {
```

Comment:
nit: you don't need `else` clause here, you can just write `return hostIsOwner || isLocalhost()`

Reply:
In this project, I don't think we have one style convention, but if we want to enforce one style, we should do it automatically, e.g. using the ESLint rule.
https://eslint.org/docs/latest/rules/no-else-return

Reply:
PR here: https://github.com/streamlit/streamlit/pull/6205